.github/workflows/provider-updoc.yml

```diff
@@ -53,10 +59,22 @@ jobs:
           GOOGLE_APPLICATION_CREDENTIALS: sa.json
           PROVIDER_NAME: ${GITHUB_REPOSITORY#*/}
           VER_MAJOR_MINOR: v${GITHUB_REF#"refs/heads/release-"}
+          SUBPACKAGES: ${{ inputs.publish-pack }}
         run: |
           if [[ "${GITHUB_REF##*/}" == release-* ]]; then
-            echo "Publishing Docs for ${{ env.PROVIDER_NAME }}, ${{ env.VER_MAJOR_MINOR }}"
-            go run github.com/upbound/uptest/cmd/updoc@${{ env.UPTEST_VERSION }} upload --docs-dir=./docs --name=${{ env.PROVIDER_NAME }} --version=${{ env.VER_MAJOR_MINOR }} --bucket-name=bucket-marketplace-docs-production --cdn-domain=https://user-content.upbound.io
+            for s in $SUBPACKAGES; do
+              PROVIDER_PACKAGE_NAME="${{ env.PROVIDER_NAME }}-$s"
+              DOCS_DIR="./docs/family"
+                if [ $s == 'monolith' ]; then
+                  PROVIDER_PACKAGE_NAME="${{ env.PROVIDER_NAME }}"
+                  DOCS_DIR="./docs/monolith"
```

Comment:
For the official providers with no families (e.g., `upbound/provider-azuread`), we will need to move their docs under to the `docs/monolith` folder. In a next iteration, we may revisit this.

Reply:
Opened a PR: https://github.com/upbound/provider-azuread/pull/75